charts/humio-operator/values.yaml

```diff
@@ -40,3 +40,6 @@ operator:
             - linux
 
 certmanager: true
+
+image: humio/humio-core:latest
+helperImage: humio/humio-operator-helper:latest
```

Comment:
Please leave the default behavior to use the hardcoded golang versions. We should never have something that defaults to using the `latest` tag.